BUILD

```diff
@@ -34,14 +34,30 @@ load("@vaticle_dependencies//distribution:deployment.bzl", "deployment")
 load(":deployment.bzl", github_deployment = "deployment")
 
 
+genrule(
+    name = "python-ffi",
+    outs = ["typedb/typedb_client_python.py"],
```

Comment:
Files and srcs should be:
`native_client_wrapper.py`
and
`//rust:typedb_client_python_wrapper`

Reply:
Renamed, but not exactly to these names.